### ISSUE TITLE
Explore cooperation with Quantopian (https://www.quantopian.com/algorithms)

### ISSUE THREAD

**Issue by tacurran** [Originate/dbg-pds-tensorflow-demo#4]
@reyfar perhaps you will find additional datasets to expand the approach

**Comment by y-f-a** [comment]
@tacurran yes - @stefansavev-o has been doing some preliminary exploration in quantopian. We'll be following that up.

**Comment by stefansavev-o** [comment]
yes, we can do this.
they have their library for back testing which might be useful to utilize in two ways:
- pds data uses the backtesting library
- the algorithms and findings we find, we implement them in this library and run on quantopian as well as locally on pds
